view/newPerspective/utils.js

```diff
@@ -249,38 +249,51 @@ function getValuesObject(request, getPerspectiveName) {
     lens: {}, // includes library
   };
 
-  return request('/v1/perspectives')
-  .then((res) => {
-    valuesObj.persNames = res.body.map((perspective) => perspective.name).sort();
+  return Promise.all([
+    request('/v1/perspectives'),
+    request('/v1/lenses'),
+    request('/v1/subjects?fields=isPublished,absolutePath,tags'),
+    request('/v1/aspects?fields=isPublished,name,tags')
+  ])
+  .then((responses) => {
+    const perspectives = responses[0].body;
```

Comment:
perspectives has to be loaded in order to populate the dropdown on the main page, so we should get that response back and populate the data into that component, WITHOUT waiting for lenses & subjects & aspects to come back (since they're not needed until user clicks to edit a perspective or create a new perspective)